src/Compilers/Core/Portable/Text/SourceHashAlgorithm.cs

```diff
@@ -22,4 +22,11 @@ public enum SourceHashAlgorithm
         /// </summary>
         Sha256 = 2,
     }
+
+    internal static class SourceHashAlgorithmUtils
+    {
+#pragma warning disable CA1802 // Use literals where appropriate
+        public static readonly SourceHashAlgorithm DefaultHashAlgorithm = SourceHashAlgorithm.Sha256;
```

Comment:
Can we name this something like `DefaultNonCryptoHash`? I want to differentiate where we are using a digest for cryptographic purposes and where we're just trying to get a content hash.

Reply:
That's already in the name of the type `SourceHashAlgorithm`, isn't it?